requirements/dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
--r test-requirements.txt
 -r extra-ml-requirements.txt
+-r test-requirements.txt
 -r lint-requirements.txt
 -r doc-requirements.txt
```

Comment:
Can we gain experience that we should put larger req file in front to make it complete faster ?